src/main/java/com/pubnub/api/models/consumer/objects_api/member/PNUUID.java

```diff
@@ -11,30 +11,50 @@
 import java.util.Map;
 
 @RequiredArgsConstructor
-public abstract class PNUUID {
+public class PNUUID {
```

Comment:
Why it's not abstract anymore? We introduce new way of creating this PNUUID which is not what it used to be. Is the plan to change it?

Reply:
It should be abstract.